lua/ulx/modules/sh/freeze.lua

```diff
@@ -0,0 +1,35 @@
+CATEGORY_NAME = "CFC"
+
+local function freezeProps( callingPlayer, targetPlayers ) 
+    local entities = ents.GetAll()
+    local entCount = 0
+    local entCounts = {}
+    for _, ply in pairs( targetPlayers ) do
+        entCounts[ply] = 0
+    end
+
+    for _, ent in pairs( entities ) do
+        local owner = ent:CPPIGetOwner()
+        if entCounts[owner] then
+            local canFreeze = not (ent:IsWeapon() or ent:GetUnFreezable() or ent:IsPlayer())
+            local physicsObj = ent:GetPhysicsObject()
+            if IsValid( physicsObj ) and canFreeze then
+                physicsObj:EnableMotion( false )
+                physicsObj:Sleep()
+                entCount = entCount + 1
+                entCounts[owner] = entCounts[owner] + 1
+            end
+        end
+    end
+    timer.Simple( engine.TickInterval() + callingPlayer:Ping()/(999), function()
```

Comment:
get rid of the timer and if you're concerned about print order  try using 
```lua
ULib.tsay( ply, "HELLO WORLD", true )
```
that should use the same system ulx uses to print so the messages will retain their order

Reply:
the true parameter is 
`wait - *(Optional, defaults to false)* Wait one frame before posting. (Useful to use from things like chat hooks)`
fancyLogAdmin uses true